mongodbatlas/resource_mongodbatlas_privatelink_endpoint_serverless.go

```diff
@@ -182,7 +183,7 @@ func resourceMongoDBAtlasPrivateLinkEndpointServerlessDelete(ctx context.Context
 		Refresh:    resourcePrivateLinkEndpointServerlessRefreshFunc(ctx, conn, projectID, instanceName, endpointID),
 		Timeout:    d.Timeout(schema.TimeoutDelete),
 		MinTimeout: 5 * time.Second,
-		Delay:      3 * time.Second,
+		Delay:      5 * time.Second,
```

Comment:
just curious, why 5 seconds delay?

Reply:
@Zuhairahmed just making it consistent across privatelink_endpoint resources.